CMakeLists.txt

```diff
@@ -45,10 +45,11 @@ endif()
 
 if(NOT "${GIT_TAG}" STREQUAL "v_${VERSION}")
     if(GIT_FOUND)
-        execute_process(COMMAND "${GIT_EXECUTABLE}" show --no-patch --format=%cs HEAD
+        execute_process(COMMAND "${GIT_EXECUTABLE}" show --no-patch --format=%ci HEAD
             WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-            OUTPUT_VARIABLE GIT_DATE
+            OUTPUT_VARIABLE GIT_DATE_FULL
             OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
but why later GIT_DATE still being used?

Reply:
In this method full date via %ci obtained in GIT_DATE_FULL, then in SUBSTRING it cutt in GIT_DATE

Reply:
Add commit with use only one variable. But I has not systems with old git and can't test in it.